k8s/openebs-operator.yaml

```diff
@@ -214,6 +221,13 @@ spec:
         # This is supported for openebs provisioner version 0.5.3-RC1 onwards
         #- name: OPENEBS_MAYA_SERVICE_NAME
         #  value: "maya-apiserver-apiservice"
+        livenessProbe:
+          exec:
+            command:
+            - pgrep
+            - ".*provisioner"
+          initialDelaySeconds: 300
```

Comment:
Can Initialdelayseconds be shortened (60 or less)? otherwise looks good to me.

Reply:
same as above